README.md

```diff
@@ -2,10 +2,16 @@
 
 Puppet module to set up Pulp 3. The primary goal of the maintainers is to set up Pulp 3 as part of Katello installation, but there's no reason it couldn't be used elsewhere.
 
+It assumes package repositories are present on the system. For convenience there is a `pulpcore::repo` class which installs from [yum.theforeman.org](https://yum.theforeman.org/pulpcore) (built from [pulpcore-packaging](https://github.com/theforeman/pulpcore-packaging)) but users can obviously use other sources. Installation from pip is not supported.
+
 ## Support policy
 
-The module provides no guarantee for multiple versions. Whenever a version is dropped, the major version is increased. All supported versions are listed.
+The module provides no guarantee for multiple versions. Whenever a version is dropped, the major version is increased.
```

Comment:
I think it could benefit to be more specific here in each mentions of versions which versions we are talking about: module version or Pulpcore version.